Improve the BugTracker

demhademha opened the pull request:
This pull request addresses the following
* To combat the spam on this repo, notify that a person's issue will be first closed and then upon   further violations, the user can be blocked from the BugTracker entirely 
* add tick boxes, which first direct a user to check the frequently asked questions on the checkra1n website and then to check for duplicate issues on this repo
* ask the user if they're on the  latest version of checkra1n 
* refer to host systems as macOS and linux, rather than hackintosh and vm

demhademha committed:
a bug form as good as a jailbreak

demhademha committed:
not just yet

demhademha committed:
there are these () and then these []

worawat0 commented:
A

itspolly reviewed (approved):
Looks good to me! 👍

Siguza commented:
I don't like the checkboxes tbh. I feel like people will just tick those either way, and we can infer that information from the rest of the issue anyway. The other change looks fine to me, apart from the fact that it should be `its OS version`.

moselhemailyy commented:
a

demhademha committed:
add the suggestions made by the checkra1n team

demhademha commented:
Hi @Siguza  I've fixed the grammatical error, and as per your suggestion, I got rid of the tick boxes and added bullet points which suggest to read the FAQ and to first check other issues. I know that you can tell by the contents of an issue, but sometimes it can serve as a psychological reminder to just check.

fahad12346 commented:
> This pull request addresses the following
> 
> * To combat the spam on this repo, notify that a person's issue will be first closed and then upon   further violations, the user can be blocked from the BugTracker entirely 
> 
> * add tick boxes, which first direct a user to check the frequently asked questions on the checkra1n website and then to check for duplicate issues on this repo
> 
> * ask the user if they're on the  latest version of checkra1n 
> 
> * refer to host systems as macOS and linux, rather than hackintosh and vm 
> 
> 

[14 ./4 checkra1n]()

cannujBarfu0negjiz commented:
.github/ISSUE_TEMPLATE/bug_report.md

Chibibowa commented:
A hackintosh is fine tbh. The worst you can have is an usb that doesn't work because it's not mapped properly. Tho there will always be at least one that will work. Especially now with OpenCore, a hackintosh is really close to a native mac. Long gone are the days of Clover being clunky and sometimes borked.

Mengsae commented on .github/ISSUE_TEMPLATE/bug_report.md:
```diff
@@ -7,12 +7,18 @@ assignees: ''
+**If this form is left empty, or the relevant sections are left empty, then your issue will be closed without any questions. Further failure to appropriately fill in the sections will result in you being blocked from the Checkra1n BugTracker**
+**before you submit an issue, make sure that you have checked the following:**
+
+ *[frequently asked questions](https://checkra.in)
```
2019-2022

Alshahran replied:
https://checkra.in/releases/0.12.3-beta

jhavisalmillon replied:
Ok
